doc/nyan/engine_types.md

```diff
@@ -75,9 +79,9 @@ Animation(): # or Sprite() ?
                  # contains all frames. # WIP5: naming scheme?
 
 #	metainfo : file # WIP6: how to read `animation'? describes, where in the big picture `animation' which frames are; whether to potentially mirror them.
-	                # possibilities: 1. $renameMe is a regular file: description of `animation' (syntax?)
-	                # 2. replace `renameMe' by more fields that describe `animation' (ugly imo, but maybe the best way)
-	                # 3. remove `renameMe':
+	                # possibilities: 1. `metainfo' is a regular file: description of `animation' (syntax?)
+	                # 2. replace `metainfo' by more fields that describe `animation' (ugly imo, but maybe the best way)
+	                # 3. remove `metainfo':
```

Comment:
we can either keep the current `docx` format or add a new one to just contain the frame positions and ids. i wouln't store information about the texture itself in nyan, the implicit reference to `filename.anim` or something might be better. the current `docx` approach is very simple and works well, we should load it on demand though and not on launch.